Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -135,6 +135,12 @@ open class NavigationMapView: UIView {
     @objc dynamic public var routeDurationAnnotationColor: UIColor = .routeDurationAnnotationColor
     @objc dynamic public var routeDurationAnnotationSelectedTextColor: UIColor = .selectedRouteDurationAnnotationTextColor
     @objc dynamic public var routeDurationAnnotationTextColor: UIColor = .routeDurationAnnotationTextColor
+
+    /**
+     List of Mapbox Maps font names to be used for any symbol layers added by the Navigation SDK.
+     These are used for features such as Route Duration Annotations that are optionally added during route preview.
+     See https://docs.mapbox.com/ios/maps/api/6.3.0/customizing-fonts.html for more information about server-side fonts.
```

Comment:
Use Markdown formatting:

```suggestion
     This property affects the font applied to the route duration annotations that the `showRouteDurations(along:)` method adds to the map view. [The map view selects the font](https://docs.mapbox.com/ios/maps/api/6.3.0/customizing-fonts.html) in the same manner as any other symbol layer that you add to the style.
```